src/libs/ajax/AzureStorage.ts

```diff
@@ -1,4 +1,5 @@
 import _ from 'lodash/fp'
+import { Ajax } from 'src/libs/ajax'
```

Comment:
This creates a circular dependency, which doesn't seem great. But other files are doing it also, so I guess it's ok?

Reply:
It's not creating a circular dependency. 

`Ajax` itself is a function, and is only used in a function in this file, so neither need each other for initialization. If this were done with two static objects instead, it would be a problem, since neither would be able to initialize without the other.